content/tracing/visualization/trace.md

```diff
@@ -52,8 +55,25 @@ When [submitting your traces][1] you can add attributes in the `meta` parameter.
 
 {{< img src="tracing/visualization/trace/trace_error_formating.png" alt="Error Formating" responsive="true" >}}
 
+{{% /tab %}}
+{{% tab "Host Info" %}}
+
+View the host information related to the trace including host tags and graphs marking the time of the trace.
```

Comment:
r/`and graphs marking the time`/`and graphs around the time of..` 

(i'm not really sure about this one but if i recall correctly host graphs are around the span timestamp)